pkg/autoscaler/autoscaler/pdplan_autoscaler.go

```diff
@@ -128,6 +131,127 @@ func (am *autoScalerManager) updateAutoscalingClusters(tac *v1alpha1.TidbCluster
 }
 
 func (am *autoScalerManager) createAutoscalingClusters(tc *v1alpha1.TidbCluster, tac *v1alpha1.TidbClusterAutoScaler, groupsToCreate []string, groupPlanMap map[string]pdapi.Plan) error {
-	// TODO in next PR
-	return nil
+	var errs []error
+	for _, group := range groupsToCreate {
+		plan := groupPlanMap[group]
+		component := plan.Component
+
+		if !checkAutoscalingComponent(tac, component) {
+			continue
+		}
+
+		resource, ok := tac.Spec.Resources[plan.ResourceType]
+		if !ok {
+			errs = append(errs, fmt.Errorf("unknown resource type %v for group %s tac[%s/%s]", plan.ResourceType, plan.Labels[groupLabelKey], tac.Namespace, tac.Name))
+			continue
+		}
+		requestsResourceList := corev1.ResourceList{
+			corev1.ResourceCPU:    resource.CPU,
+			corev1.ResourceMemory: resource.Memory,
+		}
+
+		limitsResourceList := corev1.ResourceList{
+			corev1.ResourceCPU:    resource.CPU,
+			corev1.ResourceMemory: resource.Memory,
+		}
+
+		autoTcName, err := genAutoClusterName(tac, component, plan.Labels, resource)
+		if err != nil {
+			errs = append(errs, err)
+			continue
+		}
+
+		autoTc := &v1alpha1.TidbCluster{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      autoTcName,
+				Namespace: tc.Namespace,
+				Labels: map[string]string{
+					label.AutoInstanceLabelKey:     tac.Name,
```

Comment:
Suggest adding a label for the original TC.

Reply:
added